drivers/filesystems/udfs/Include/wcache_lib.cpp

```diff
@@ -1128,10 +1128,11 @@ WCacheUpdatePacket(
                             (PVOID)WCacheSectorAddr(block_array, Lba0),
                             BS) != BS);
             }
-            if(mod)
+            else {
```

Comment:
This change should actually not be necessary (by the principle of minimal changes in 3rd-party code), even if it's true that:
```
if(mod) { /* something */} if(!mod) { /* something else */ }
```
is equivalent to
```
if(mod) { /* something */ } else { /* something else */ }
```

Reply:
Sorry. Probably you are right, that it should be an "if". I just changed this that reactos was building on my PC. I will add just the brackets arround.